apps/omg_watcher_info/test/omg_watcher_info/db/txoutput_test.exs

```diff
@@ -46,6 +46,36 @@ defmodule OMG.WatcherInfo.DB.TxOutputTest do
     assert utxo.amount == big_amount
   end
 
+  @tag fixtures: [:phoenix_ecto_sandbox, :alice]
```

Comment:
instead of adding this test please write a cabbage test

Reply:
🙆‍♂  will move to cabbage